workflows/nf_aggregate/tests/main.nf.test

```diff
@@ -10,7 +10,7 @@ nextflow_pipeline {
 
         when {
             params {
-                outdir     = "results"
+                outdir = "results"
```

Comment:
^^ We should probably change tags to 'workflow' if you're doing this. Also, as a workflow-level test it needs inputs etc